tests/_convert/snapshots/convert_duplicate_definitions_read_before_write.py.txt

```diff
@@ -0,0 +1,33 @@
+import marimo
+
+app = marimo.App()
+
+
+@app.cell
+def _():
+    x = 1
+    return (x,)
+
+
+@app.cell
+def _(x):
+    x
+    return
+
+
+@app.cell
+def _(x):
+    x
+    x_1 = 2
+    x_1
+    return (x_1,)
```

Comment:
This is interesting. In the other test the expected output is:

```py
x; x = 2; x
```

Looks like it's unrolled here.